packages/victory-chart/src/helper-methods.js

```diff
@@ -17,9 +17,6 @@ function getAxisProps(child, props, calculatedProps) {
   const axis = child.type.getAxis(childProps);
   const crossAxis = childProps.crossAxis === false ? false : true;
```

Comment:
Since we're calculating the full domain in `VictoryAxis` I think we can also easily let it handle `crossAxis` and `orientation` by itself now.